scripts/lint-staged/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "../tsconfig.scripts.json",
```

Comment:
**chore:** you need to register this in `tsconfig.scripts.json` and exclude in `scripts/tsconfig.json` to make it work as expected